international-labour-organization.csl

```diff
@@ -137,70 +141,84 @@
         <date-part name="year"/>
       </date>
       <choose>
-        <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <if type="book" match="any">
           <group prefix=" " delimiter=". " suffix=".">
-            <text macro="title"/>
+            <text macro="title" font-style="italic"/>
+            <text variable="collection-title"/>
             <text macro="edition"/>
             <text macro="editor"/>
+            <text variable="volume" form="short" text-case="capitalize-first"/>
             <text macro="publisher"/>
+            <text macro="pages"/>
+            <text variable="number-of-pages"/>
           </group>
         </if>
         <else-if type="paper-conference" match="any">
-          <text macro="title" font-style="italic" prefix=" " suffix=","/>
+          <text variable="title" quotes="false" font-style="normal" font-weight="normal" prefix=" " suffix=","/>
           <group prefix=" " delimiter=" ">
             <text macro="editor"/>
             <text variable="container-title" font-style="italic" suffix="."/>
             <text variable="collection-title" suffix=","/>
-            <text macro="edition"/>
+            <text macro="edition" suffix=" "/>
```

Comment:
no need for the suffix here -- it's covered by the delimiter in the group above.